app/src/main/java/ch/epfl/sdp/game/StartGameController.java

```diff
@@ -0,0 +1,5 @@
+package ch.epfl.sdp.game;
+
+public interface StartGameController {
+    void start();
+}
```

Comment:
initGame should start the game loop so is there a reason you absolutely need an interface like this? I know it's not easy to look into the codebase and use stuff that's already there but it will pay off since the assistants complained about having too much classes and interfaces.

Reply:
Here the start in StartGameController means inits the game area, enemy, and item box, after that initGame. The reason I need this interface is that now game area, enemy and item box are all location-based, so I can only init them after fetching the location(instead of the default 0, 0) for the first time in the GoogleLocationFinder